src/components/Popover/index.jsx

```diff
@@ -1,300 +1,57 @@
-  /** JSX - Acts as a positioning reference for the popover and triggers active state */
-  trigger: PropTypes.oneOfType([PropTypes.node, PropTypes.element]).isRequired,
-
-  /**
-   * JSX - Content to place in the positioned popover container. The popover container
-   * does not provide any default styling; content should be styled with background,
-   * borders, and shadows as appropriate.
-   */
-  children: PropTypes.oneOfType([PropTypes.node, PropTypes.element]).isRequired,
-
-  /**
-   * What mode of interaction triggers the active/inactive state of the popover.
-   *
-   * `hover` - popover opens on trigger hover/focus
-   *
-   * `click` - popover opens on trigger click
-   *
-   * `controlled` - enables "fully controlled" mode in which the popover is only active
-   * when the `isOpen` prop is set to `true`. Use `onUserDismiss` to handle dismissal
-   * events from user interaction.
-   */
-  interactionMode: PropTypes.oneOf(VALID_INTERACTION_MODES),
-
-  /** Controlls active state of popover when in fully controlled interaction mode */
+  appendTo: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
+  children: PropTypes.node,
+  /** Element that popover is positioned on */
+  triggerElement: PropTypes.node,
+  placement: PropTypes.oneOf(['auto', 'top', 'right', 'bottom', 'bottom-start', 'left']),
   isOpen: PropTypes.bool,
-
-  /**
-   * Callback fired when user takes an action to dismiss the popover.
-   * (e.g. ESC press, clickikng outside, etc.)
-   * Useful for updating `isOpen` in controlled mode.
-   * Function is invoked with the DOM event as the argument.
-   */
-  onUserDismiss: PropTypes.func,
-
-  /** disables portaling the popover to `document.body` */
-  disablePortal: PropTypes.bool,
-
-  /**
-   * Vertical position preference of popover content.
-   * `top` for example, will place the popover content above the trigger.
-   */
-  position: PropTypes.oneOf(VALID_POSITIONS),
-
-  /** Horizontal alignment preference of popover content relative to trigger */
-  alignment: PropTypes.oneOf(VALID_ALIGNMENTS),
-
-  /** Offset distance from trigger. */
-  distance: PropTypes.number,
-
-  /** Delay in milliseconds for popover to trigger (hover mode only) */
-  delay: PropTypes.number,
-
-  /** Name of transition for popover content */
-  transitionName: PropTypes.oneOf(['GrowFast']),
-
-  /** Callback called when popover opens */
-  onOpen: PropTypes.func,
-
-  /** Callback called when popover closes */
-  onClose: PropTypes.func,
-
-  /** When popover opens, this element should freeze scrolling. When the popover closes, the scrollability of this element should resume. */
-  disableScrollRef: validRef,
-
-  /** When the Popover detects scroll events from this ref, the popover should close. */
-  closeOnScrollRef: validRef,
-
-  /** Ref to forward to the content container of the popover */
-  ref: validRef,
+  onClickOutside: PropTypes.func,
+  trigger: PropTypes.oneOf(['click', 'mouseenter focus']),
```

Comment:
is `mouseenter focus` supposed to be a single value?

Reply:
also, how does someone create a controlled popover? Is there a `trigger` value in tippy that allows something like the below example?

`<Popover isOpen={isOpenFromOuterState} onUserDismiss={() => {setIsOpenFromOuterState(false)}} ... />`

Reply:
Yup 😁 

<img width="988" alt="Screen Shot 2020-12-29 at 4 03 30 PM" src="https://user-images.githubusercontent.com/708820/103313894-68407c00-49ef-11eb-8d45-08f150037927.png">

Reply:
Nice - so now I'm wondering if we should tuck these options behind our facade component...

`triggerType: PropTypes.oneOf(['manual', 'hover', 'click'])`

In this case, "hover" would map to `mouseenter focus`.

Either way, I think we need to add `manual` as a valid prop value here

Reply:
Makes sense. I sort of wonder if adding `isOpen` makes it manual trigger automatically (as the above is tippy, but we're dealing with the react implementation): I'll research and let you know, but definitely agree with the facade.

Reply:
```
<Popover
  appendTo={() => {}}
  isOpen
  onClickOutside={function noRefCheck() {}}
  placement="bottom-start"
  theme="default"
  trigger="click"
  triggerElement={<button onClick={function noRefCheck() {}}>Open popover</button>}
>
  <div className="padding--all">
    <h3 className="type--head3">
      Look at me
    </h3>
    <p>
      <em>
        i am the popover
      </em>
    </p>
  </div>
</Popover>
```

I changed the story so that it's more apparent, but you can control `isOpen` which makes the manual trigger irrelevant. I'm assuming that's for non React implementations. I am going to add the `hover` facade though.